manifests/backend/resources.pp

```diff
@@ -1,152 +1,25 @@
-# @summary Default sensu resources
+# @summary Define sensu resources
```

Comment:
why does this class stick around?

Reply:
I wanted to define the resources in separate class to avoid cluttering up `sensu::backend`.  Will be even more important if using iteration as that will take up more lines of code.